.pre-commit-config.yaml

```diff
@@ -16,6 +16,21 @@ repos:
               | ^poetry/core/_vendor
           )
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+      - id: mypy
+        additional_dependencies:
+          - types-requests
+        files: |
```

Comment:
Let's codify these paths in `pyproject.toml` ala-poetry so that editor-invoked mypy will respect them.